app/containers/Web3Alerts/Paused.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Wrapper } from './styles';
+
+const Paused = () => (
+  <Wrapper theme="warning">
+    <h2>Contracts under maintenance</h2>
+    <p>
+      Currently we are unable to send transactions. Please try later
```

Comment:
Maybe we can use prismic to customize the message? So we can display ETA and similar information. Can be a separate task though

Reply:
Makes sense. And agree about a separate task.